TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/timeline/UserListTimeline.java

```diff
@@ -0,0 +1,144 @@
+package com.ybsystem.tweetmate.fragments.timeline;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.fragment.app.Fragment;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.adapters.recycler.UserListRecyclerAdapter;
+import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.models.entities.Column;
+import com.ybsystem.tweetmate.models.entities.twitter.TwitterUserList;
+import com.ybsystem.tweetmate.utils.ExceptionUtils;
+import com.ybsystem.tweetmate.utils.ToastUtils;
+
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers;
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.observers.DisposableObserver;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import twitter4j.ResponseList;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.UserList;
+
+import static com.ybsystem.tweetmate.models.enums.ColumnType.*;
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class UserListTimeline extends TimelineBase {
+
+    private Column mColumn;
+
+    public Fragment newInstance(Column column) {
+        Bundle bundle = new Bundle();
+        bundle.putSerializable("COLUMN", column);
+        setArguments(bundle);
+        return this;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate
+        View view = inflater.inflate(R.layout.fragment_timeline, container, false);
+
+        // Init
+        mColumn = (Column) getArguments().getSerializable("COLUMN");
+        mScrollLoad = true;
+
+        // Set view
+        setRecyclerView(view);
+        setRecyclerAdapter(new UserListRecyclerAdapter());
+        setUltraPullToRefresh(view);
+        mPtrFrame.setEnabled(false);
+
+        return view;
+    }
+
+    @Override
+    protected void loadTweet(boolean isPullLoad, boolean isClickLoad) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 19 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS3776)